doc/classes/ProjectSettings.xml

```diff
@@ -1607,6 +1607,12 @@
 		</member>
 		<member name="rendering/vulkan/descriptor_pools/max_descriptors_per_pool" type="int" setter="" getter="" default="64">
 		</member>
+		<member name="rendering/vulkan/forward_mobile/color_buffer_format" type="int" setter="" getter="" default="0">
```

Comment:
I dont think this should be an option, mobile should always use 10 bits.

Reply:
That would make things easier but it does mean giving up alpha unless we add a separate R8 buffer for that as the ARM guys suggest. I'm just not sure how to bind that to `frag_color.a`